README.md

```diff
@@ -1 +1,186 @@
-# Budget-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+    - [🚀 Live Video](#live-video)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget app <a name="about-project"></a>
+
+**Budget app** is an app created using Ruby on Rails, which is about creating some groups and transactions between them. In short, it is an app that helps you keep track of your budget
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Use ruby gems as software packages system.**
+- **Install Ruby on Rails framework.**
+- **Understand Rails RESTful design and router.**
+- **Use controllers to handle requests and render empty views.**
+- **Use params from browser request in a safe way.**
+- **Use preprocessed html file with embedded Ruby code.**
+- **Use layouts and templates for shared content.**
+- **Use database migration files to maintain database schema.**
+- **Use validations for models.**
+- **Secure app from n+1 problems.**
+- **Understand what ORM is.**
+- **Write SQL queries with ActiveRecord.**
+- **Set up associations between models.**
+- **Build a webapp that requires the user to log in.**
+- **Use devise gem for authentication.**
+- **Limit access to webapp resources based on authorization rules.**
+- **Analyze in writing why you have made a coding choice using one structure over another.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budgett-3abs.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE VIDEO -->
+
+## 🚀 Live Video <a name="live-video"></a>
+
+- [Live Video Link](https://drive.google.com/file/d/1Bz207uE94ipB6YXCPpy6EODpYJS6Innq/view?usp=drive_link)
+
```

Comment:
- [ ] Great job including the right video link. However, I think you forgot to make it accessible as a result I wasn't able to view it so kindly fix that as well :wink: 
- Unaccessible Page
![image](https://github.com/dieum512/Budget-app/assets/102699958/5040f063-7e51-46a7-99e7-e46252cd0280)

Reply:
Done :ok_hand:

Reply:
addressed